### PR TITLE
Rewrite with ES6 syntax

### PR DESCRIPTION
**Pull request by neighborhood999**

### CODE REVIEW

File: src/views/components/settings/index.js [hunk 1]
```diff
@@ -78,34 +101,34 @@ var SettingsComponent = React.createClass({
         Remote.getCurrentWindow().setAlwaysOnTop(newPreference);
       }
     });
-  },
+  }
 
-  _makeLanguageOptions: function(languages, defaultLanguage) {
+  _makeLanguageOptions (languages, defaultLanguage) {
```
Comment (EragonJ) [inline src/views/components/settings/index.js]: no space

File: src/views/components/settings/index.js [hunk 2]
```diff
@@ -78,34 +101,34 @@ var SettingsComponent = React.createClass({
...
       option.value = language.lang;
       option.selected = (language.lang === defaultLanguage);
       select.add(option);
     });
-  },
+  }
 
-  _makeTopRankingOptions: function(countryData, defaultCountryCode) {
-    var select = this.refs.topRankingSelect;
+  _makeTopRankingOptions (countryData, defaultCountryCode) {
```
Comment (EragonJ) [inline src/views/components/settings/index.js]: no space

**Comment by neighborhood999** [comment]
Sorry, forget upload this file.
When I had been refactoring and getting this error, seems error come from `L10nSpan`, after we are refactoring need fix this issue.

**Comment by EragonJ** [comment]
jshinit related comments can be also removed btw.

**Comment by EragonJ** [comment]
ok plz file a new issue for that, I noticed that from inspector too. Overall the patch looks ok, plz check again and if things are going well, feel free to merge!

**Comment by neighborhood999** [comment]
Test script doesn't have syntax check, it doesn't affect testing in travis.ci. Yep! we can remove it.

**Comment by neighborhood999** [comment]
I already removed all the `jshint` comment.

**Pull request merged**